res/values/strings.xml

```diff
@@ -358,6 +358,8 @@
         <item quantity="other">%d new messages</item>
     </plurals>
     <string name="chat_no_messages_hint">Send message to %1$s:\n\n• It is okay if %2$s does not use Delta Chat.\n\n• Delivering the first message may take a while and may be unencrypted.</string>
```

Comment:
this was the old string, it can be deleted once the UI pick up the new one (some string cleanup is needed anyway at some point :)